docs/blog/posts/2024-03-28-run-gpu.md

```diff
@@ -47,66 +40,76 @@ chmod +x Miniconda3-latest-Linux-x86_64.sh
 conda config --set auto_activate_base false
 ```
 
-See <a href="https://uit.stanford.edu/service/oak-storage/" target="_blank">this page</a> to learn more about Oak and to purchase space for your lab. 
+You can save this file to your **home** directory and call it `install_miniconda.sh` for example. When you run the script from your home directory, it will ask to provide a path where to install miniconda.
+
+Make sure to paste your **Oak path** instead of the default home installation (for example, `/oak/stanford/projects/<your-lab>/<$USER>/miniconda`). We want to avoid creating multiple conda environments in the home directory because, with machine learning, data science, and deep learning packages, it is possible to run out of space after setting up just a few environments. Therefore, use Oak or another path where you have large amount of space.
 
-To run the script from home,
+Refer to [this page](https://uit.stanford.edu/service/oak-storage/){:target="_blank"} to learn more about Oak and to purchase space for your lab.
 
-```bash
+To run the script from home directory:
+
+```title="Terminal Command"
 sh install_miniconda.sh
 ```
 
-After miniconda is installed, add path to conda bin to your bash profile so that `conda` and `python` executables are found:
+After miniconda is installed, add the path to conda bin to your bash profile so that `conda` and `python` executables are found:
+
+```bash linenums="1" title=".bash_profile"
 
-```bash
-# add path to miniconda
+# Add path to miniconda
 export PATH=$PATH:/oak/stanford/projects/<your-lab>/<$USER>/miniconda3/bin
 ```
 Source bash profile to apply changes:
 
-```bash
+```title="Terminal Command"
 source ~/.bash_profile
 ```
-
-#### Create conda env
+### Create Conda Environment
```

Comment:
We recommend against using conda pretty much everywhere, I think the default sherlock packages might be able to do this out of the box?  That's what I verified when I wrote the Sherlock page: http://127.0.0.1:8000/_user_guide/sherlock/#deep-learning-on-gpu

Reply:
I thought about that too. I guess the Sherlock page I referenced was not authored by you. Since it still mentioned the conda environment, I included it in the this page. Since the content is largely the same, should we just remove this blog page? My primary contribution to this page was rewriting the Colab section, but I'm not sure how helpful it is. If it is useful, we cam create a separate page for Colab. Thoughts?

Reply:
Let me know if the Colab page looks good or need any edits.